src/views/ModelPlan/ReadOnly/_components/Sidenav/index.tsx

```diff
@@ -55,13 +55,15 @@ const SideNav = ({
 
       // Calculate all the things
       const distanceFromTopOfPage =
-        top +
+        Math.round(top) +
         window.scrollY -
         filterBannerHeight -
         navBarHeight -
         marginOfScrollElement;
 
-      window.scroll(0, distanceFromTopOfPage);
+      if (window.scrollY >= distanceFromTopOfPage) {
```

Comment:
Might just be useful to slap a comment on this check explaining what it does (I assume this is basically checking if the view has been scrolled _past_ the `distanceFromTopOfPage`, i.e. if you've scrolled past the header, navbar, etc.?